addons/revolutionaries/CfgVehicles_O_Russian.hpp

```diff
@@ -143,6 +143,103 @@ class TACU_Revolutionaries_U_O_Russian_Autorifleman: TACU_Revolutionaries_U_O_Ru
     editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
 };
 
+class TACU_Revolutionaries_U_O_Russian_FigureHeadUA: TACU_Revolutionaries_U_O_Russian_Rifleman01 {
+    displayName = "Figure Head (Unarmed)";
+    uniformClass = "CUP_U_B_USMC_FROG3_WMARPAT";
+    linkedItems[] = {DEFAULT_ITEMS, "CUP_H_USMC_CAP_WDL"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "CUP_H_USMC_CAP_WDL"};
+    backpack = "";
+    weapons[] = {"Throw", "Put"};
+    respawnWeapons[] = {"Throw", "Put"};
+    magazines[] = {""};
+    respawnMagazines[] = {""};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Russian_FigureHead: TACU_Revolutionaries_U_O_Russian_FigureHeadUA {
+    displayName = "Figure Head";
+    weapons[] = {"CUP_hgun_TaurusTracker455_gold", "Throw", "Put"};
+    respawnWeapons[] = {"CUP_hgun_TaurusTracker455_gold", "Throw", "Put"};
+    magazines[] = {mag_6("CUP_6Rnd_45ACP_M")};
+    respawnMagazines[] = {mag_6("CUP_6Rnd_45ACP_M")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Russian_Leader: TACU_Revolutionaries_U_O_Russian_Rifleman01 {
+    displayName = "Leader";
+    uniformClass = "CUP_U_B_USMC_FROG1_WMARPAT";
+    linkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "CUP_H_USMC_BOONIE_PRR_WDL"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "CUP_H_USMC_BOONIE_PRR_WDL"};
+    weapons[] = {"TACU_Revolutionaries_W_CZ805_GL", "Throw", "Put"};
+    respawnWeapons[] = {"TACU_Revolutionaries_W_CZ805_GL", "Throw", "Put"};
+    magazines[] = {mag_6("CUP_30Rnd_556x45_G36"), mag_6("1Rnd_HE_Grenade_shell")};
+    respawnMagazines[] = {mag_6("CUP_30Rnd_556x45_G36"), mag_6("1Rnd_HE_Grenade_shell")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
```

Comment:
```suggestion
    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
    headgearList[] = {
        "CUP_H_USMC_BOONIE_PRR_WDL", 1
    };
```

Also same as before with the facewear - if you don't want unit to have facewear, add a CBA_facewearList entry.